_posts/2018/2018-03-20-novy-predseda-pardubickeho-kraje.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+category: CLANKY
+title: Piráti si zvolili nového předsedu Pardubického kraje 
+date: 2018-03-20T10:10:00+01:00  
+tags: Pardubice PKS Volby 
+author: Andrej Ramašeuski
+image: assets/img/miscellaneous/pivo-v-holicich.jpg #751x422
+
+---
+
+Koncem minulého týdne si zvolili členové krajského sdružení České pirátské strany 
+svého nového předsedu. Stal se jím [Mgr. Jiří Jansa]({{ site.url }}/lide/jiri-jansa/). 
+Dosavadní předseda 
+Pavel Štěpánek se vzdal funkce, protože se stal krajským koordinátorem.
+
+Nominaci do volby obdržel Jiří Jansa a dosavadní krajský koordinátor 
+Daniel Lebduška, výsledný rozdíl mezi nimi byl jen 2 hlasy. 
+Daniel Lebduška k tomu říká: 
+
+> "Gratuluji Jirkovi ke zvolení. Jsem vlastně rád, že to tak dopadlo. 
+Teď se můžu, kromě své rodiny a zaměstnání, plně věnovat Chrudimskému komunálu."
+
+Nový krajský předseda Jiří Jansa vyrůstal v Chrudimi a žije v Pardubicích. 
+Je mu 33 let a nyní pracuje třetím rokem v pojišťovně v sekci pojistné matematiky, 
+předtím působil tři roky v menší firmě na pozici makronalytika měnových kurzů. 
+Vystudoval Fakultu sociálních věd na Karlově univerzitě, obor Ekonomické teorie. 
+V průběhu studia strávil rok ve švédském městě Umeå. 
+Členem Pirátské strany je pár měsíců, aktivním příznivcem již zhruba 6 let.
+
+> "Své zvolení jsem úplně nečekal, jelikož jsem nekandidoval s tolik politickým programem" 
+
+říká ke svému zvolení. Ten obsahuje mimo jiné koordinovanou podporu otevřenému 
+software a technologiím, školení programování pro veřejnost nebo 
+podporu neziskovým komunitním akcím a studentským grantům.
+
+> "V ČR teprve doháníme západní země v transparentnosti státní správy. 
+Po celém světě ale nyní probíhá další průmyslová revoluce v podobě 
+robotizace a automatizace, která bude mít zanedlouho dalekosáhlé
+důsledky do života všech lidí. Společnost jako celek se musí 
+na nastupující změny připravovat a jednou z možných cest by mohl 
+být postupný přechod k otevřeným systémům, výrobkům, software, 
+hardware nebo vytváření menších propojených komunit pomocí moderních 
+technologií. Rádi bychom i v našem kraji tuto vlnu co nejvíce podporovali," 
+
+uzavírá Jiří Jansa s tím, že krajská buňka pirátské strany 
+je připravena podporovat projekty v tomto duchu i z řad zcela apolitické veřejnosti.
```

Comment:
toto mi prijde nepresne, protoze jsme to delali doposud. Spis bych to videl jako "budeme pokracovat v nastavene politici podpory ...."

Reply:
tak tady bych do autorskeho textu nezasahoval

Reply:
zkonzultovat s nim to lze ne?

Reply:
Zkonzultovat to s nim lze ne?

Reply:
tak zkonzultij :)

Reply:
myslim si, ze vedel co rika